init_db.sh

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
-cd data
+
```

Comment:
Is there a reason this init script is in this PR? Kunal just merged this in yesterday and I was wondering why this had to change

Reply:
this totally isn't necessary, it was just my hack around to make the init script work on my machine since my data directory has weird permissions. 
I can remove it if you want.

Reply:
Ah I see, I mean it works for me as you have it here, so I'm good with keeping it if it allows your device to use it too